### ISSUE TITLE
Bug Report: Girlfriend's position changes when the stage refreshes

### ISSUE THREAD

**Issue by Lethrial** [FunkinCrew/Funkin#3483]
### Issue Checklist

- [X] I have properly named the issue
- [X] I have checked the issues/discussions pages to see if the issue has been previously reported

### What platform are you using?

Itch.io (Downloadable Build) - Windows

### If you are playing on a browser, which one are you using?

None

### Version

0.5.0

### Context (Provide images, videos, etc.)

When restarting on a song, whether by death or through the pause menu, the Girlfriend character's position on the stage changes. The amount the position changes varies per character it seems. With Girlfriend for example, her position is shifted very slightly up.

https://github.com/user-attachments/assets/88661ac8-eef1-49cf-be64-aa2de51a3acf

The issue is most apparent on Blazin', where Nene moves up and left on restart, which causes Darnell and Pico to look like they've moved off the screen as the camera refocuses on her new position. (This one in particular was also reported in https://github.com/FunkinCrew/Funkin/issues/3199 but I figured it'd be better to make this a seperate issue)

https://github.com/user-attachments/assets/c7a3d364-3bae-489c-9994-148a8a4205d8

### Steps to reproduce (or crash logs, errors, etc.)

Restart or die and retry on any song.

**Comment by Lethrial** [comment]
Oops, didn't notice that Boyfriend also moves in the first clip; I guess it's not a GF specific issue then

**Comment by Hundrec** [comment]
Weird bug! Not sure if the Week 3 offsets have been reported before, but it should definitely be addressed.

**Comment by OrbyyOrbinaut** [comment]
This appears to be a global problem (noticable in some areas more than others)

**Comment by Lethrial** [comment]
Yeah, the issue seems to have settled quite a lot in 0.5.1 but the bug is still present in places, including Blazin

**Comment by Lasercar** [comment]
Does this still happen in 0.5.3?

**Comment by Hundrec** [comment]
Tried a bit on Philly and Mall/Mall Erect, didn't see it happen

Will need further testing and confirmation!

**Comment by Hundrec** [comment]
It's so strange that the bricks in Stress (Pico Mix) move and need to be manually reset in the script, so I think the root cause is still present

**Comment by Hundrec** [comment]
Let's get a new report on this if it still happens in 0.8.1 (where we have flixel-animate)
